### DISCUSSION TITLE
Talisman wallet

### DISCUSSION THREAD

**Discussion by andreibadea20** [rainbow-me/rainbowkit#818] [Ideas]
What do you think about adding talisman wallet?

**Comment by chidg** [comment]
Hi, engineer from Talisman here. We would like to develop an integration for RainbowKit. We would be happy to make a PR if that would be considered.

Talisman is a multichain web3 wallet developed initially for the Polkadot ecosystem, but now also offering full support for EVM networks. More about Talisman: https://talisman.xyz

Thanks for the mention @andreibadea20. Look forward to hearing back from RainbowKit team.
